# MODEL1310300000: MODEL1310300000

## Installation

Download this repository, and install with distutils

`python setup.py install`

Or, install using pip

`pip install git+https://github.com/biomodels/MODEL1310300000.git`

To install a specific version (in this example, from the 2014-09-16 BioModels release)

`pip install git+https://github.com/biomodels/MODEL1310300000.git@20140916`

## Usage

Importing the model package.

`import MODEL1310300000 as model`

Get the SBML string from the model

`print model.sbmlString`

If [python-libsbml](https://pypi.python.org/pypi/python-libsbml) bindings are
installed, the libsbml.SBMLDocument object is also accessible

`model.sbml`


# Model Notes


Flahaut2013 - Genome-scale metabolic model of L.lactis (MG1363)

Genome-scale metabolic model for _Lactococcus lactis_ MG1363 and its
application to the analysis of flavor formation.

This model is described in the article:

[Genome-scale metabolic model for Lactococcus lactis MG1363 and its
application to the analysis of flavor
formation.](http://identifiers.org/pubmed/23974365)

Flahaut NA, Wiersma A, van de Bunt B, Martens DE, Schaap PJ, Sijtsma L, Dos
Santos VA, de Vos WM

Applied Microbiology and Biotechnology. 2013; 97(19):8729-8739

Abstract:

Lactococcus lactis subsp. cremoris MG1363 is a paradigm strain for lactococci
used in industrial dairy fermentations. However, despite of its importance for
process development, no genome-scale metabolic model has been reported thus
far. Moreover, current models for other lactococci only focus on growth and
sugar degradation. A metabolic model that includes nitrogen metabolism and
flavor-forming pathways is instrumental for the understanding and designing
new industrial applications of these lactic acid bacteria. A genome-scale,
constraint-based model of the metabolism and transport in L. lactis MG1363,
accounting for 518 genes, 754 reactions, and 650 metabolites, was developed
and experimentally validated. Fifty-nine reactions are directly or indirectly
involved in flavor formation. Flux Balance Analysis and Flux Variability
Analysis were used to investigate flux distributions within the whole
metabolic network. Anaerobic carbon-limited continuous cultures were used for
estimating the energetic parameters. A thorough model-driven analysis showing
a highly flexible nitrogen metabolism, e.g., branched-chain amino acid
catabolism which coupled with the redox balance, is pivotal for the prediction
of the formation of different flavor compounds. Furthermore, the model
predicted the formation of volatile sulfur compounds as a result of the
fermentation. These products were subsequently identified in the experimental
fermentations carried out. Thus, the genome-scale metabolic model couples the
carbon and nitrogen metabolism in L. lactis MG1363 with complete known
catabolic pathways leading to flavor formation. The model provided valuable
insights into the metabolic networks underlying flavor formation and has the
potential to contribute to new developments in dairy industries and cheese-
flavor research.

This model is hosted on [BioModels Database](http://www.ebi.ac.uk/biomodels/)
and identified by:
[MODEL1310300000](http://identifiers.org/biomodels.db/MODEL1310300000) .

To cite BioModels Database, please use: [BioModels Database: An enhanced,
curated and annotated resource for published quantitative kinetic
models](http://identifiers.org/pubmed/20587024) .

To the extent possible under law, all copyright and related or neighbouring
rights to this encoded model have been dedicated to the public domain
worldwide. Please refer to [CC0 Public Domain
Dedication](http://creativecommons.org/publicdomain/zero/1.0/) for more
information.


